Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,26 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const totalHourWork = tasks.map(min => min.duration).reduce((hour, min) => hour + min / 60, 0);
+
+  const totalSalary = totalHourWork * hourlyRate;
+  const totalSalaryInDoller = totalSalary.toLocaleString('en-US', {
```

Comment:
You actually don't need to do this toLocaleString here. Just rounding to two decimals is enough.